tools-v2/internal/error/error.go

```diff
@@ -69,7 +70,7 @@ func (ce *CmdError) ToError() error {
 	if ce.Code == CODE_SUCCESS {
 		return nil
 	}
-	return fmt.Errorf(ce.Message)
+	return errors.New(ce.Message)
```

Comment:
The effect is the same, why is it modified like this?

Reply:
As described in #2801, `fmt.Errorf` treats the input string as a format string. This can be problematic when the error message contains a `%` character (For example a web URL with non-ASCII characters). In such cases, the % character is mistakenly treated as the beginning of a format verb, leading to incorrect output. Using `errors.New` can fix this problem.

To illustrate, the following code snippet
```go
msg := "GET failed: https://example.com?msg=hello%20world"
err1 := fmt.Errorf(msg)
err2 := errors.New(msg)
fmt.Println(err1.Error())
fmt.Println(err2.Error())
```
outputs
```
GET failed: https://example.com?msg=hello%!w(MISSING)orld
GET failed: https://example.com?msg=hello%20world
```